tests/e2e/utils/src/flows/merchant.js

```diff
@@ -210,11 +211,53 @@ const merchant = {
 		} );
 	},
 
-  openImportProducts: async () => {
+  	openImportProducts: async () => {
 		await page.goto( WP_ADMIN_IMPORT_PRODUCTS , {
 			waitUntil: 'networkidle0',
 		} );
 	},
+
```

Comment:
We should be adding a comment block on new (or changed) functions.

Reply:
Oh gotcha! I know we haven't traditionally added comments in here, so moving forward it sounds like we'll want to, is that right? So maybe we'd format them something like this?

```js
/**
 * Opens the WordPress updates page in Dashboard -> Updates
 */
```

Reply:
I would use the same format as we have been using in the `api` package.

Reply:
Sounds good. I've added comments to the new functions. I wonder if we should create an issue to go back in and add in docs for the existing `shoppe` and `merchant` flows as well.

Reply:
> I wonder if we should create an issue to go back in and add in docs for the existing shoppe and merchant flows as well.

I think we can do this per file when we are already making changes to the file on another issue. That's the way PHP coding standards issues are handled.

Reply:
Ah gotcha, that sounds good!